.github/workflows/publish.yml

```
@@ -0,0 +1,47 @@
+name: publish
+'on':
+  push:
+    branches:
+      ktp-docs-ci
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y pandoc git
+        pip3 install -r requirements-doc.txt
+        pip3 install scikit-learn-intelex
+        cd doc
+        source set_version.sh
+        echo Generating version $DOC_VERSION
+    - name: build docs
+      run: |
+        cd doc
+        chmod +x build-doc.sh
+        ./build-doc.sh
+        mv _build/scikit-learn-intelex ../../output

```

```suggestion
        doc/build-doc.sh
        mv doc/_build/scikit-learn-intelex ../output
```
Sorry since I am not so knowledgeable on the docs side. If build-doc.sh doesn't have the right permissions, we can set those in the repo to what is necessary (does it hurt to store build-doc.sh as 755 instead)?